src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -227,6 +228,64 @@ export class MoveToLineEndTerminalAction extends BaseSendTextTerminalAction {
 	}
 }
 
+export class SendSequenceTerminalCommand extends Command {
+	public static readonly ID = TERMINAL_COMMAND_ID.SEND_SEQUENCE;
+	public static readonly LABEL = nls.localize('workbench.action.terminal.sendSequence', "Send Custom Sequence To Terminal");
+
+	public runCommand(accessor: ServicesAccessor, args: any): void {
+		const terminalInstance = accessor.get(ITerminalService).getActiveInstance();
+		if (!terminalInstance) {
+			return;
+		}
+		const text = args.text.split(' ');
+		var terminalText;
+		for (let i = 0; i < text.length; i++) {
+			terminalText = ' ' + prepareTerminaText(text[i]);
```

Comment:
Why the extra space?

Reply:
We need to add a space at the end or at the beginning (to separate text or any command). 
If we keep a space at the end, it will have an extra space(near terminal prompt) in the new line after execution is over.